src/components/__tests__/__snapshots__/publicAPI.spec.js.snap

```diff
@@ -7723,9 +7723,7 @@ Map {
             "action": Object {
               "type": "func",
             },
-            "description": Object {
-              "type": "string",
-            },
+            "description": [Function],
```

Comment:
FYI this is happening because this prop is now wrapped in a `deprecate` function. I think it is unavoidable.